src/actions/GraphDataActions.ts

```diff
@@ -93,6 +94,34 @@ export const GraphDataActions = {
       if (namespace.name === 'istio-system') {
         restParams['includeIstio'] = true;
       }
+
+      // Some appenders are expensive so only specify an appender if needed.
+      let appenders: string = 'dead_node,sidecars_check,istio';
+
+      if (!node) {
```

Comment:
If you want this can be removed because the server-side is smart enough to ignore the unused_node appender for a node detail graph,

Reply:
I would think server-side should not be so smart - how does it know that the client doesn't want this appender? I would think the server should run the appenders the client asks for.

Reply:
I don't think the server-side is not being unreasonably smart.  I think skipping this appender is just part of the node-detail graph definition.  Anyway, you can leave this in here if it makes you happy.